packages/botonic-plugin-contentful/bin/migrations/content-types/daterange.ts

```diff
@@ -0,0 +1,41 @@
+import Migration, { ContentType, MigrationContext } from 'contentful-migration'
+
+import { createFieldName } from '../factories'
+
+function createFromTo(
+  dateRange: ContentType,
+  id: string,
+  name: string,
+  helpText: string
+): ContentType {
+  dateRange
+    .createField(id)
+    .name(name)
+    .type('Date')
+    .required(true)
+    .localized(false)
+  dateRange.changeFieldControl('from', 'builtin', 'datePicker', {
+    format: 'timeZ',
+    ampm: '24',
+    helpText: helpText,
+  })
+  return dateRange
+}
+
+module.exports = function (migration: Migration, _: MigrationContext) {
```

Comment:
I guess that that JS will not complain if we drop it, but since it's part of  contentful cli callback, I think it's useful to leave it as documentation in case we need it (I think it's a TS good practice not to trim a callback arguments list)